operators/oadp-operator/0.4.0/manifests/oadp-operator.v0.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,506 @@
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - velero.io
+          resources:
+          - '*'
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - list
+          - get
+          - create
+          - delete
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - use
+          resourceNames:
+          - privileged
+          - velero-privileged
+        - apiGroups:
+          - ""
+          resources:
+          - secrets
+          - configmaps
+          - pods
+          - services
+          - serviceaccounts
+          verbs:
+          - list
+          - get
+          - create
+          - delete
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - route.openshift.io
+          resources:
+          - routes
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+      - serviceAccountName: velero
+        rules:
+        - apiGroups:
+          - '*'
+          resources:
+          - '*'
+          verbs:
+          - '*'
+        - nonResourceURLs:
+          - '*'
+          verbs:
+          - '*'
+        - apiGroups:
+          - security.openshift.io
+          resources:
+          - securitycontextconstraints
+          verbs:
+          - use
+          resourceNames:
+          - privileged
+          - velero-privileged
+    strategy: deployment
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: false
+    type: AllNamespaces
+  keywords:
+  - velero
+  - openshift
+  - oadp
+  links:
+  - name: OADP Operator
+    url: https://oadp.konveyor.io/
+  maintainers:
+  - name: Dylan Murray
+    email: dymurray@redhat.com
+  - name: Shubham Dilip Pampattiwar
+    email: spampatt@redhat.com
+  maturity: beta
+  provider:
+    name: Red Hat
+  replaces: oadp-operator.v0.3.0
```

Comment:
This line is not required if updating with semver mode https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#semver-mode